Gemfile

```diff
@@ -195,7 +197,7 @@ gem 'rubyzip', require: 'zip'
 gem 'nokogiri', '>= 1.8.1'
 
 # Polyglot support
-gem 'coursemology-polyglot'
+gem 'coursemology-polyglot', '0.3.8'
```

Comment:
why lock to `0.3.8`?

Reply:
This gem is we updating ourselves, and last time it's not locked, this gem got updated to 0.3.5 and created an error of cannot detect Python 3.12 object..

We need to update that polyglot gem first to denote 0.3.8 as the latest before we open the lock

Reply:
It shouldn't update to a lower version, should check what happened, don't lock so that it will auto bump to latest.

Reply:
I got the reason why this happened
I used the `bundle update` since this is what's suggested by Rails Dev for upgrading to the newer version, and now that I checked around, it's already an existing issue that running that command can lower the version, as mentioned here in https://github.com/rubygems/bundler/issues/1266.

It should be taken note for the other CM developers also, that if want to use this command, need to proceed with caution..

Reply:
The issue mentions it's possible if there are conflicting dependencies specified, but `coursemology-polyglot` shouldn't be specific in any other gem as far as I can tell, so it's strange.  
Nevermind don't need to focus on this, just keep it unlocked and ensure it doesn't downgrade.